content/guides/frameworks/laravel/development-setup.md

```diff
@@ -0,0 +1,342 @@
+      php-fpm:
+        condition: service_started  # Wait for php-fpm to start
+
+  php-fpm:
+    # For the php-fpm service, we will create a custom image to install the necessary PHP extensions and setup proper permissions.
+    build:
+      context: .
+      dockerfile: ./docker/php-fpm/Dockerfile
+      args:
+        UID: ${UID}
+        GID: ${GID}
+        XDEBUG_ENABLED: ${XDEBUG_ENABLED}
+        XDEBUG_MODE: ${XDEBUG_MODE}
+        XDEBUG_HOST: ${XDEBUG_HOST}
+        XDEBUG_IDE_KEY: ${XDEBUG_IDE_KEY}
+        XDEBUG_LOG: ${XDEBUG_LOG}
+        XDEBUG_LOG_LEVEL: ${XDEBUG_LOG_LEVEL}
+    env_file:
+      # Load the environment variables from the Laravel application
+      - .env
+    user: "${UID}:${GID}"
+    volumes:
+      # Mount the application code for live updates
+      - ./:/var/www
+    networks:
+      - laravel
+    depends_on:
+      postgres:
+        condition: service_started  # Wait for postgres to start
+
+  workspace:
+   # For the workspace service, we will also create a custom image to install and setup all the necessary stuff.
+    build:
+      context: .
+      dockerfile: ./docker/workspace/Dockerfile
+      args:
+        UID: ${UID}
+        GID: ${GID}
+        XDEBUG_ENABLED: ${XDEBUG_ENABLED}
+        XDEBUG_MODE: ${XDEBUG_MODE}
+        XDEBUG_HOST: ${XDEBUG_HOST}
+        XDEBUG_IDE_KEY: ${XDEBUG_IDE_KEY}
+        XDEBUG_LOG: ${XDEBUG_LOG}
+        XDEBUG_LOG_LEVEL: ${XDEBUG_LOG_LEVEL}
+    tty: true  # Enables an interactive terminal
+    stdin_open: true  # Keeps standard input open for 'docker exec'
+    env_file:
+      - .env
+    volumes:
+      - ./:/var/www
+    networks:
+      - laravel
+
+  postgres:
+    image: postgres:16
+    ports:
+      - "${POSTGRES_PORT}:5432"
+    environment:
+      - POSTGRES_DB=${POSTGRES_DATABASE}
+      - POSTGRES_USER=${POSTGRES_USERNAME}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    networks:
+      - laravel
+
+  redis:
+    image: redis:alpine
+    networks:
+      - laravel
+
+networks:
+  laravel:
+
+volumes:
+  postgres-data:
+```
+
+> [!NOTE]
+> Ensure you have an `.env` file at the root of your Laravel project with the necessary configurations (e.g., database and Xdebug settings) to match the Docker Compose setup.
+
+### Running Your Development Environment
+
+To start the development environment, use:
+
+```console
+$ docker compose -f compose.yaml up --build -d
+```
+
+This command will build and start all the required services, including PHP, Nginx, and the PostgreSQL database. You can now access your Laravel application at `http://localhost/`.
+
+### Summary
+
+By setting up a Docker Compose environment for Laravel development, you ensure that your development setup is consistent and easily reproducible. This makes it easier for you and your team to collaborate on the same project, without worrying about differences in local environments.
+
+<div id="compose-lp-survey-anchor"></div>
```

Comment:
```suggestion
```

Reply:
@dvdksn should I remove this `<div id="compose-lp-survey-anchor"></div>` from the other pages too? What impact does it have?